nltk/test/bleu.doctest

```diff
@@ -12,3 +12,18 @@ If the candidate has no alignment to any of the references, the BLEU score is 0.
 ...     [1],
 ... )
 0
+
+This is an implementation of the smoothing techniques
+for segment-level BLEU scores that was presented in
+Boxing Chen and Collin Cherry (2014) A Systematic Comparison of
+Smoothing Techniques for Sentence-Level BLEU. In WMT14.
+http://acl2014.org/acl2014/W14-33/pdf/W14-3346.pdf
+>>> from nltk.translate.bleu_score import sentence_bleu,SmoothingFunction
+
+
+>>> sentence_bleu(
+...     ['It is a place of quiet contemplation .'.split()],
+...     'It is .'.split(),
+...     smoothing_function=SmoothingFunction().method4,
+... )*100
+4.426730592500287
```

Comment:
@anonymityBoy 
I would very much advise using for example `4.4267...` instead of the exact value of `4.426730592500287`.
The former will test correctly with any float that starts with (or rounds to, I forget) `4.4267`. This allows for results which are pretty much exactly correct, but differ due to some inconsistencies in float operations. The latter will likely cause some tests to arbitrarily fail due to these inconsistencies in float operations, e.g.:
```python
>>> 1.2 * 1.5
1.7999999999999998
```

Reply:
Good point. I format the output.